libs/ledger-live-common/src/families/celo/specs/createActivateVoteMutation.ts

```diff
@@ -0,0 +1,44 @@
+import invariant from "invariant";
+import { getCryptoCurrencyById, parseCurrencyUnit } from "../../../currencies";
+import { MutationSpec } from "../../../bot/types";
+import type { CeloAccount, Transaction } from "../types";
+
+const currency = getCryptoCurrencyById("celo");
+const minimalAmount = parseCurrencyUnit(currency.units[0], "0.001");
+
+export const createActivateVoteMutation = (): MutationSpec<Transaction> => ({
```

Comment:
⚠️  there are not even a single test on this mutation

Reply:
@gre  correct me if I'm wrong, but there is an inherit test that's run on each mutation that verifies that the transaction was broadcasted successfully - that said, the most critical functionality is, in fact, tested.  This pattern exists in Cosmos bot tests, among others.

Reply:
@andyhass Thanks for your feedback.
What @gre mentions here is the point of adding a `test:` field within every mutation, or even at the global spec level.
When there is no test defined, what the bot tests is indeed that the broadcast succeeds, which is the minimum, but there are often many additional checks that can be done around the transaction to validate more things.
Probably not all of our specs do it (yet), but you can check many examples for instance the one for cosmos does have additional tests defined:
- https://github.com/LedgerHQ/ledger-live/blob/2da52a7b570bb4e20ba2d40735e1d5ef082c3f31/libs/ledger-live-common/src/families/cosmos/specs.ts#L68
- https://github.com/LedgerHQ/ledger-live/blob/2da52a7b570bb4e20ba2d40735e1d5ef082c3f31/libs/ledger-live-common/src/families/cosmos/specs.ts#L102
etc.